### ISSUE TITLE
Fix /config/edition

### ISSUE THREAD

**Issue by LeadSongDog** [internetarchive/openlibrary#6353]
@mekarpeles 
The /config/edition page has long been a mess. Previous issues internetarchive/openlibrary#142 internetarchive/openlibrary#725 seem to have been inadvertently closed without a fix. Regular users seem to lack permission to edit the file and no indication is shown of how to get it corrected. User "Administrator" has made a ton of edits with each appending one blank line. Many of the URL formats shown no longer work, or redirect. Many more identifiers simply don’t show a corresponding URL.

Without working hot links for external identifiers, they will rarely be useful. With them, many rich libraries’ catalogues (and often content) become more accessible.

- [x] internetarchive/openlibrary#7826
- [x] internetarchive/openlibrary#7825
- [x] internetarchive/openlibrary#7824
- [x] internetarchive/openlibrary#8294
- [x] internetarchive/openlibrary#7785

**Comment by cdrini** [comment]
CC @onnotasler who has been creating lots of great specific issues about what needs to be fixed. I think that's what we need to make progress on this ; basically a list of fixes.

@LeadSongDog could you create an issue for the blank line thing? Any other specific issues on that page that need to be fixed?

**Comment by onnotasler** [comment]
Indeed, I created several bugs with broken identifiers. I have not checked all identifiers yet, though.

#7826 
#7825 
#7824 
#7792 
#7785

**Comment by cdrini** [comment]
Thank you @onnotasler ! I'm going to assign this one to you and promote those issues to a checklist here :)

**Comment by jimchamp** [comment]
Edition identifiers are now defined in a [version-controlled file](https://github.com/jimchamp/openlibrary/blob/master/openlibrary/plugins/openlibrary/config/edition/identifiers.yml), so issues requesting new identifiers should no longer be accidentally closed.

The extra line being added on config edit likely still remains, but is tracked by #7840.

Can this be closed?
